src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/Testing/Common.cs

```diff
@@ -0,0 +1,20 @@
+using Datadog.Trace.Ci;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing
+{
+    internal static class Common
+    {
+        static Common()
+        {
+            TestTracer = Tracer.Instance;
```

Comment:
What if we replace the Tracer entirely with a new one like this:

```suggestion
            TestTracer = new Tracer(Tracer.Instance.Settings);
```

Reply:
My only concern is that by doing this will cause disconnected traces If the test calls something that we have an integration for, because the integration will be using the `Tracer.Instance` isn't?  And one of the goals for the CIApp team is to have distributed traces for the tests for integration tests.

Reply:
Oh I see, yes the traces would be disconnected then. I didn't realize distributed tracing inside integration tests was a goal, thanks for clarifying 👍🏼